image/installer/bin/functions/config_pipeline

```diff
@@ -103,14 +103,38 @@ function config_pipeline() {
         ;;
       *)
         prompt_for_input 'Custom Subscription Channel' MAS_APP_CHANNEL_MANAGE
-        prompt_for_input 'Custom Catalog Source' MAS_APP_SOURCE_SAFETY "ibm-mas-manage-operators"
+        prompt_for_input 'Custom Catalog Source' MAS_APP_SOURCE_MANAGE "ibm-mas-manage-operators"
         ;;
     esac
     export MAS_APP_CHANNEL_MANAGE; export MAS_APP_SOURCE_MANAGE
   else
     export MAS_APP_SOURCE_MANAGE=''; export MAS_APP_CHANNEL_MANAGE=''
   fi
 
+  # MSO can only be installed before MAS 8.8
```

Comment:
I thought we agreed the other day that we'd simply support optimizer only and ignore mso for the scope of the automation?

Reply:
@durera that is true! what happened is that it was not a big deal to add that for the sake of completeness 🙂. It's working anyway, also in fvt. What do you think to keep it? Either is good for me.

Btw, forgot to add install plan selection in case of optimize, will include that in the next commit